### DISCUSSION TITLE
`prose` compatibility with Daisy's classes

### DISCUSSION THREAD

**Discussion by 4l1fe** [saadeghi/daisyui#2648] [Q&A] [answered]
Hey, i came from Bootstrap, it's much more handy to use Daisy than pure Tailwind, great job.

I'm trying to change link colors with Daisy's classes but when it comes to editing an article where  the class `prose` is set up they don't work.

I do as follow accordingly to the [Tailwind doc section](https://tailwindcss.com/docs/typography-plugin#element-modifiers)

```html
<head>
<script src="https://cdn.tailwindcss.com?plugins=forms,typography,aspect-ratio,line-clamp""></script>
<link href="https://cdn.jsdelivr.net/npm/daisyui@4.4.19/dist/full.min.css" rel="stylesheet" type="text/css" />
</head>

...

    <div class="flex flex-row justify-center">
        <div class="basis-1/2 mt-4 max-w-none prose prose-a:link prose-a:link-error prose-a:link-hover">
        {% block body %}
        {% endblock %}
        </div>
   </div>
```

I guess you know what the modifier `prose-a` is. 

So, is it possible to combine both classes? I can't avoid using `prose` and am going to use it more extensively in the future/

**Comment by saadeghi** [comment, marked as answer]
It works: https://play.tailwindcss.com/1yDqZfPGhD  
But not with CDN. 

Because with CDN, you're not installing daisyUI, so Tailwind CSS can not dynamically generate daisyUI colors.

Currently Tailwind CSS CDN file only supports their own first party plugins. Not third party plugins like daisyUI.

Let me know if you have a question.

**Reply by 4l1fe** [reply]
Thx. I haven't used Tailwind and stuff as javascript files yet, only css files for now, and am not experienced in the front-end dev.
